src/triage/util/pickling.py

```diff
@@ -0,0 +1,23 @@
+import pickle
+
+
+def can_pickle(obj):
+    """Check whether or not an object can be pickled by attempting to pickle it to
+    a BytesIO
+
+    Since it pickles to an in-memory object, be wary about using on huge objects!
+    """
+    try:
+        pickle.dumps(obj)
+    except Exception as e:
+        # Implementation note:
+        # The Python docs for pickle claim that all picking errors will result
+        # in a PickleError subclass. In testing, this did not seem to be the case:
+        # different types of non-picklable objects raised totally different types of errors.
+        # The only commonality was that their exception message contained the word 'pickle'
+        # So this seems brittle, but maybe less brittle than any alternatives.
+        if "pickle" in str(e):
+            return False
+        else:
+            raise
```

Comment:
I really appreciate this attempt to differentiate pickling errors from others but, especially considering how finicky it is to positively identify pickling errors, and considering the extreme unlikeliness that an exception here is due to anything *but* that the object cannot be pickled, why not just test that any exception is raised?

We could, instead, use the ForkingPickler, and get even closer to what the multiprocessing module tries to do. Regardless, if what it wants to do, (pickle), raises an exception, we know that "it can't be pickled."